### PR TITLE
Handle repo rename in build scripts and add GH action caching

### PR DESCRIPTION
**Pull request by knolleary**
## Description

The build scripts copy the FF docs from `../flowforge/docs`. This PR updates the scripts to work with `../flowfuse/docs` and fall back to `../flowforge/docs` otherwise. This eases the transition and doesn't break anyone who hasn't rebuild their local environment.

It also enables caching of the `_site/img` directory in the GH action. This *should* speed up the time it takes to run the `test` action (after the initial run completes to populate the cache).

### CODE REVIEW

**Comment by knolleary** [comment]
Once the initial run of the [`Test` action](https://github.com/FlowFuse/website/actions/runs/6379117161) completes for this PR (in about 30 minutes), I think we should be able to rerun it to observe the caching in action...

File: package.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
         "build:js": "terser -c -m -o _site/js/cc.min.js node_modules/vanilla-cookieconsent/dist/cookieconsent.js src/js/cookies.js",
         "build": "cross-env NODE_ENV=production npm-run-all clean build:js --parallel prod:*",
         "clean:dev": "cross-env npx del-cli '_site/!(img)' 'src/docs/**/*.md' && cross-env npx mkdirp '_site/js'",
```
Comment (Pezmc) [inline package.json:14]: This is now redundant, it was originally added for the exclusion of the img dir, they can both just use `clean` now

**Review by Pezmc — COMMENTED** [review]
Assuming the build re-runs faster and the redundant package script is removed, 👍

**Comment by Pezmc** [comment]
Second run took only 1m 46s (down from 30 mins); I've just enabled npm caching too, which brought the build down to 1m 30s, a notable improvement.

File: package.json (hunk 1 above)
Comment (Pezmc) [reply package.json:14]: Not quite true, we need to clean the docs directly locally: https://github.com/FlowFuse/website/pull/969

**Pull request merged**